modules/exploits/apple_ios/browser/safari_jit.rb

```diff
@@ -41,7 +41,12 @@ def initialize(info = {})
         'DefaultTarget' => 0,
         'DefaultOptions' => { 'PAYLOAD' => 'apple_ios/armle/meterpreter_reverse_tcp' },
         'Targets' => [[ 'Automatic', {} ]],
-        'DisclosureDate' => '2016-08-25'
+        'DisclosureDate' => '2016-08-25',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
```

Comment:
I really doubt this is crash safe. This is JIT in Safari followed by a shellcode memory write and kernel exploit. Most of the time in my experience this is unstable at best and highly temperamental. Is there a reason that we believe this is crash safe that I'm not aware of?

Same comment also applies to the REPEATABLE_SESSION part here.